unstructured_inference/inference/layout.py

```diff
@@ -246,12 +246,16 @@ def get_elements_with_detection_model(
         if self.ocr_mode == "individual_blocks":
             ocr_layout = None
         elif self.ocr_mode == "entire_page":
-            ocr_data = pytesseract.image_to_data(
-                self.image,
-                lang=self.ocr_languages,
-                output_type=Output.DICT,
-            )
-            ocr_layout = parse_ocr_data(ocr_data)
+            ocr_layout = None
+            try:
+                ocr_data = pytesseract.image_to_data(
+                    self.image,
+                    lang=self.ocr_languages,
+                    output_type=Output.DICT,
+                )
+                ocr_layout = parse_ocr_data(ocr_data)
+            except pytesseract.pytesseract.TesseractError:
+                logger.warning("TesseractError: Skipping page")
```

Comment:
add `, exc_info=True` so there is a chance at debugging these issues?
actually same for the other try/catch in https://github.com/Unstructured-IO/unstructured-inference/pull/165/files